pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -725,12 +719,22 @@ func applyLoadBalancer(c *cluster.Cluster, lb *networking.LoadBalancerSettings,
 		c.LbPolicy = cluster.Cluster_CLUSTER_PROVIDED
 		c.ClusterDiscoveryType = &cluster.Cluster_Type{Type: cluster.Cluster_ORIGINAL_DST}
 	default:
-		c.LbPolicy = defaultLBAlgorithm()
+		ApplyDefaultLoadBalancer(c, lb)
 	}
 
 	ApplyRingHashLoadBalancer(c, lb)
 }
 
+// ApplyDefaultLoadBalancer will set the DefaultLBPolicy and create an LbConfig if used in LoadBalancerSettings
+func ApplyDefaultLoadBalancer(c *cluster.Cluster, loadbalancer *networking.LoadBalancerSettings) {
```

Comment:
prefer calling` applyDefaultLoadBalancerAlgorithm`. ApplyDefaultLoadBalancer is too wide

Reply:
I renamed to applyDefaultSimpleLoadBalancer as that func is intended to be used to set other lb settings in future if they come up - it is not just lb algo.